frontend/public/components/utils/status-icon.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { Icon } from 'patternfly-react';
+
+export const StatusIcon: React.FunctionComponent<StatusIconProps> = ({status}) => {
+
+  if (!status){
+    return <span>-</span>;
+  }
+
+  switch (status) {
+
+    case 'Pending':
+      return <span><Icon type="fa" name="hourglass-half" /> {status}</span>;
```

Comment:
Does the layout still work if we use `React.Fragment` instead of `span`? (Here and below)

```suggestion
      return <React.Fragment><Icon type="fa" name="hourglass-half" /> {status}</React.Fragment>;
```

Reply:
The layout works fine without the span, however the enzyme test that tries to read the text associated with the icon fails. It seems it's a known issue with enzyme but unless the StatusIcon component is wrapped in an element such as a span, it cannot read the text to check it's value. Should I remove that check from the test? or keep the span?